docs/modeling/projects.md

```diff
@@ -69,4 +72,6 @@ File definitions are created and stored for each element of a project:
 * `<decision-table-name>.xml` is the format that decision table definitions are stored in.
```

Comment:
```suggestion
* `<decision-table-name>-extensions.xml` is the format that decision table definitions are stored in.
```

Reply:
@mmartinadanx I think that the change you propose is in the following line (the json file not the xml)

Reply:
You are right. Thanks!